fuzz/cms.c

```diff
@@ -34,11 +34,19 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
         return 0;
 
     in = BIO_new(BIO_s_mem());
+    if (in == NULL)
+        return 0;
+
     OPENSSL_assert((size_t)BIO_write(in, buf, (int)len) == len);
     cms = d2i_CMS_bio(in, NULL);
     if (cms != NULL) {
         BIO *out = BIO_new(BIO_s_null());
 
+        if (out == NULL) {
+            BIO_free(in);
```

Comment:
The `cms` should be freed here too

Reply:
@jogme isn't it freed on a previous line?